roles/pulp_common/tasks/install_pip.yml

```diff
@@ -93,7 +93,7 @@
 
 - name: Include pulp-rpm prereq role
   include_role:
-    name: pulp_rpm_prerequisites
+    name: pulp.pulp_installer.pulp_rpm_prerequisites
```

Comment:
I did some testing. It looks like if I want to run these roles outside of a collection against a host, or with molecule, I do not need any changes that git would conflict with other than modifying this one line to go back to pulp_rpm_prerequisites.

(The other changes would be thinks like the roles_path in my ~/.ansible.cfg, and symlinks for the roles folder in the molecule dirs).

Would you consider not changing this 1 line?

Reply:
My test results were, that ansible 2.8 desparately needs it.